src/postgraphile/__tests__/withPostGraphileContext-test.js

```diff
@@ -209,6 +209,34 @@ test('will add extra claims as available', async () => {
   ]);
 });
 
+test('will include JWT claims as jwtClaims in context callback', async () => {
+  const pgClient = { query: jest.fn(), release: jest.fn() };
+  const pgPool = { connect: jest.fn(() => pgClient) };
+  const { jwtClaims } = await withPostGraphileContext(
+    {
+      pgPool,
+      jwtToken: jwt.sign({ aud: 'postgraphile', a: 1, b: 2, c: 3 }, 'secret', {
+        noTimestamp: true,
+      }),
+      jwtSecret: 'secret',
+    },
+    context => context,
+  );
+  expect(jwtClaims).toEqual({ aud: 'postgraphile', a: 1, b: 2, c: 3 });
+});
+
+test('jwtClaims should be an empty object if there is no JWT token', async () => {
+  const pgClient = { query: jest.fn(), release: jest.fn() };
+  const pgPool = { connect: jest.fn(() => pgClient) };
+  const { jwtClaims } = await withPostGraphileContext(
+    {
+      pgPool,
+    },
+    context => context,
+  );
+  expect(jwtClaims).toEqual({});
+});
+
```

Comment:
If there’s no JWT then jwtClaims should be undefined (or null). It defaults to empty object only to make the logic of that function simpler; you’ll need to add a check e.g. `jwtClaims: hasJwt?jwtClaims:null` Into the return value of getSettingsForPgClientTransaction

Reply:
Alrighty!